camel-workitem/pom.xml

```diff
@@ -16,6 +16,21 @@
     <name>jBPM dev team</name>
   </organization>
 
+  <properties>
+    <java.module.name>org.jbpm.contrib.camel-workitem</java.module.name>
+    <maven.hibernate.dialect>org.hibernate.dialect.H2Dialect</maven.hibernate.dialect>
+    <maven.datasource.classname>org.h2.jdbcx.JdbcDataSource</maven.datasource.classname>
+    <maven.jdbc.driver.class>org.h2.Driver</maven.jdbc.driver.class>
+    <maven.jdbc.db.name/>
+    <maven.jdbc.db.port/>
+    <maven.jdbc.db.server/>
+    <maven.jdbc.driver.jar/>
+    <maven.jdbc.username>sa</maven.jdbc.username>
+    <maven.jdbc.password>sasa</maven.jdbc.password>
+    <maven.jdbc.url>jdbc:h2:tcp://localhost/${project.basedir}/target/jbpm-test;MVCC=TRUE</maven.jdbc.url>
+    <maven.jdbc.schema>public</maven.jdbc.schema>
+  </properties>
+
```

Comment:
Just curious why we need database with the upgrade, otherwise looks good.

Reply:
it is for the camel-sql tests. i can do it as part of a separate pr if you prefer it. np

Reply:
If they will be added right after, I don't see any issue with having it prepared.